packages/intl/i18n/en.json

```diff
@@ -216,6 +216,18 @@
   "serverError.title": {
     "message": "Something went wrong..."
   },
+  "share.copyLink": {
+    "message": "Copy link"
```

Comment:
We already have `actions.copyShareUrl` so maybe we could re-use it if it's already translated?

I was also thinking if it would make sense to name these `actions.<name>` as these texts are pretty generic.